botocove/cove_host_account.py

```diff
@@ -278,16 +279,28 @@ def _get_active_org_accounts(self) -> Set[str]:
         Captures all account metadata into self.account_data for future lookup
         and returns a set of account IDs in the AWS organization.
         """
-        pages = self.org_client.get_paginator("list_accounts").paginate()
+        active_accounts: Iterable[AccountTypeDef] = tqdm(
```

Comment:
It's a shame tqdm is pretty ugly by default (might look better outside of pytest?) -
```
Listing accounts  : 5it [00:00, 638.64it/s]
```
I thought this might be somewhere a spinner might be a nicer touch, but looks like that idea died on the vine in tqdm: https://github.com/tqdm/tqdm/issues/925#issuecomment-1021262212

If you're up for an experiment it might be a nice upgrade to swap to https://github.com/Textualize/rich - my criteria so far is that it has double the github stars..ish and explicitly supports spinners :)

Reply:
I like the spinner concept.

Rich has many to choose from. See [Will McGugan's Demo of Rich "spinner" animations](https://www.youtube.com/watch?v=CLkLvOmNOjc) on Youtube.

[alive-progress](https://github.com/rsalmei/alive-progress) also has plenty to choose from. The README include inspring gifs.

If we had to change dependency for the spinner, I would prefer to try alive-progress first.

Rich has many more stars, but it's a bigger framework with more use cases.

I'm judging this based on the README of each. I haven't used either yet, so happy to be shown otherwise.

I don't have time to experiment just now, but I may do so in the future.

For now would you be happy to go with tqdm's ugly and functional progress reporting?